Improve desktop version of nav bar

yieb opened the pull request.

yieb committed:
Update navbar logo to blue

yieb committed:
Add dropdown item to navbar and fix centering

yieb committed:
Update navbar

yieb commented:
Hamburger icon for mobile view appears when screen size changes now, but it does not function properly. We could remove the onClick property of the image tag while the mobile view of the navbar gets developed

heylookitsme reviewed (approved):
edit: i will write more details after i merge the rest

Commit:
Made navbar flex, apply dropdown hides when not active

- refactored a bit: dropdown -> applyDropdown, more specific naming
- consolidated menu options into navs/uls for accessibility
- made transitions a bit faster
- swapped opacity/display attributes for applyDropdown for visibility
- removed <button> wrapping element

todo:
- make applyDropdownButton an id
- clean up any more dead code
- mobile
